### PR TITLE
Process descendant filtering in event filters [serverless]

### PR DESCRIPTION
**Pull request by joepeeples**
Contributes to https://github.com/elastic/security-docs/issues/5402 by adding to the serverless docs.  Serverless twin of https://github.com/elastic/security-docs/pull/5626, which has already been reviewed & merged, so mostly just need a codeowner approval.

> [!NOTE]
> The one notable change here is updating the "Added" badge in the ESS docs to no longer mention serverless. That's because once the feature is available in serverless, we'll no longer need to call out any differences between ESS and serverless. 

Preview
- Click [the link below](https://github.com/elastic/security-docs/pull/5672#issuecomment-2273762148), then in the Security card click **View serverless docs**. 
- Navigate to: **Manage endpoint protection → Event filters**

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-1o2gjo3ea-elastic-dev.vercel.app

* Built with commit 196d684b7831608eabafe0b2e693be3ed48f8a44

> Issues? Visit #next-docs in Slack

File: docs/serverless/edr-manage/event-filters.mdx [hunk 1]
```diff
@@ -53,6 +53,12 @@ Create event filters from the Hosts page or the Event filters page.
       1. `Description`: Enter a filter description (optional).
 1. In the **Conditions** section, depending which page you're using to create the filter, either modify the pre-populated conditions or add new conditions to define how ((elastic-sec)) will filter events. Use these settings:
     1. `Select operating system`: Select the appropriate operating system.
+    1. Select which kind of event filter you'd like to create:
+    
+        * `Events`: Create a generic event filter that can match any event type. All matching events are excluded.
+        * `Process Descendants`: Create a filter that suppresses the descendant activity of a specified process. Events from the matched process will be ingested, but events from its descendant processes will be excluded.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/serverless/edr-manage/event-filters.mdx:59]: Slight tweak to start the sentence with an active command:
```suggestion
        * `Process Descendants`: Specify a process, and suppress the activity of its descendant processes. Events from the matched process will be ingested, but events from its descendant processes will be excluded.
```

File: docs/management/admin/event-filters.asciidoc [hunk 2]
```diff
@@ -42,7 +42,7 @@ image::images/event-filter.png[Add event filter flyout, 80%]
   .. `Description`: Enter a filter description (optional).
 . In the **Conditions** section, depending which page you're using to create the filter, either modify the pre-populated conditions or add new conditions to define how {elastic-sec} will filter events. Use these settings:
   .. `Select operating system`: Select the appropriate operating system.
-  .. Select which kind of event filter you'd like to create: added:[8.15.0,Coming to {serverless-full}.]
+  .. Select which kind of event filter you'd like to create: added:[8.15.0]
     * `Events`: Create a generic event filter that can match any event type. All matching events are excluded.
     * `Process Descendants`: Create a filter that suppresses the descendant activity of a specified process. Events from the matched process will be ingested, but events from its descendant processes will be excluded.
```
Comment (joepeeples) [inline docs/management/admin/event-filters.asciidoc:47]: Apply suggestion from [here](https://github.com/elastic/security-docs/pull/5672/files#r1715289035) (serverless) to ESS docs:
```suggestion
    * `Process Descendants`: Specify a process, and suppress the activity of its descendant processes. Events from the matched process will be ingested, but events from its descendant processes will be excluded.
```

File: docs/serverless/edr-manage/event-filters.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashokaditya) [reply docs/serverless/edr-manage/event-filters.mdx]: Oh this is even better!

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**